racksdb/web/ui/src/views/DatacenterDetailsView.vue

```diff
@@ -7,16 +7,41 @@ SPDX-License-Identifier: GPL-3.0-or-later -->
 <script setup lang="ts">
 import { useHttp } from '@/plugins/http'
 import { useRacksDBAPI } from '@/composables/RacksDBAPI'
-import { ref, onMounted, watch } from 'vue'
-import SearchBar from '@/components/SearchBar.vue'
+import { ref, onMounted, watch, computed } from 'vue'
 import type { Ref } from 'vue'
 import type { Datacenter, DatacenterRoom } from '@/composables/RacksDBAPI'
 import BreadCrumbs from '@/components/BreadCrumbs.vue'
+import { BarsArrowDownIcon, BarsArrowUpIcon, ChevronRightIcon } from '@heroicons/vue/24/outline'
+import { useRouter } from 'vue-router'
 
+const router = useRouter()
 const http = useHttp()
 const racksDBAPI = useRacksDBAPI(http)
 const datacenters: Ref<Array<Datacenter>> = ref([])
 const datacenterDetails: Ref<Datacenter | undefined> = ref()
+const alphabeticalOrder = ref(true)
+const input = ref('')
+const filteredItem = computed(() =>
```

Comment:
This variable holds an array of filtered rooms, it should be rename `filteredRooms`.